src/ray/util/util.h

```diff
@@ -117,6 +117,38 @@ inline int64_t current_sys_time_us() {
   return mu_since_epoch.count();
 }
 
+static std::random_device              rd;
```

Comment:
```suggestion
static std::random_device rd;
```
IIRC, our lint rule should prefer this style. Right?

Reply:
yeah, it's been detected by lint. I'll change it.